python/packages/devui/agent_framework_devui/_utils.py

```diff
@@ -124,6 +142,10 @@ def select_primary_input_type(message_types: list[Any]) -> Any | None:
     if not message_types:
         return None
 
+    for message_type in message_types:
```

Comment:
Hi @victordibia, would like to get your thoughts here, please: right now, if an executor has a union of types it can accept, and we find a `str`, we are preferring this type over other perhaps complex types. Now, for orchestrations/agent-specific executors, I am checking if it contains a `ChatMessage` type, and preferring that over others, so that it can more closely abide by the agent protocol. Thoughts?